save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/LnkUserOrganizationController.kt

```diff
@@ -0,0 +1,203 @@
+/**
+ * Controller for processing links between users and their roles in organizations:
+ * 1) to put new roles of users
+ * 2) to get users and their roles by organization
+ * 3) to remove users from organizations
+ */
+
+package org.cqfn.save.backend.controllers
+
+import org.cqfn.save.backend.security.OrganizationPermissionEvaluator
+import org.cqfn.save.backend.service.LnkUserOrganizationService
+import org.cqfn.save.backend.service.OrganizationService
+import org.cqfn.save.backend.utils.AuthenticationDetails
+import org.cqfn.save.domain.Role
+import org.cqfn.save.info.UserInfo
+import org.cqfn.save.permission.Permission
+import org.cqfn.save.permission.SetRoleRequest
+import org.cqfn.save.v1
+
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.security.core.Authentication
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.server.ResponseStatusException
+
+/**
+ * Controller for processing links between users and their roles in organizations
+ */
+@RestController
+@RequestMapping("/api/$v1/organizations/")
+class LnkUserOrganizationController(
+    private val lnkUserOrganizationService: LnkUserOrganizationService,
+    private val organizationService: OrganizationService,
+    private val organizationPermissionEvaluator: OrganizationPermissionEvaluator,
+) {
+    private val logger = LoggerFactory.getLogger(LnkUserOrganizationController::class.java)
+
+    /**
+     * @param organizationName
+     * @param authentication
+     * @return list of users with their roles, connected to the organization with [organizationName]
+     * @throws ResponseStatusException
+     */
+    @GetMapping("{organizationName}/users")
+    fun getAllUsersByOrganizationName(
+        @PathVariable organizationName: String,
+        authentication: Authentication,
+    ): List<UserInfo> {
+        val organization = organizationService.findByName(organizationName)
+            ?: throw ResponseStatusException(HttpStatus.NOT_FOUND)
+        val usersWithRoles = lnkUserOrganizationService.getAllUsersAndRolesByOrganization(organization)
+            .also { logger.debug(it.toString()) }
```

Comment:
I think it will be hard to understand where did this message comes from, when we will look into debug logs, since there is no any additional words about what is printing

Reply:
Yes, it was only for debugging. Not sure if we need such logs.